packages/hasura-auth-js/src/machines/authentication/machine.ts

```diff
@@ -687,9 +687,16 @@ export const createAuthMachine = ({
         isAutoRefreshDisabled: () => !autoRefreshToken,
         refreshTimerShouldRefresh: (ctx) => {
           const { expiresAt } = ctx.accessToken
+
           if (!expiresAt) {
             return false
           }
+
+          // This happens when either the computer goes to sleep or when Chrome descides to suspend the tab
+          if (expiresAt.getTime() < Date.now()) {
```

Comment:
I think we should have a margin, i.e., 60s. Otherwise, you can end in a situation where the token was valid in this line but on the next one it has expired.